basicLibrary/src/main/java/basicLibrary/lab3.java

```diff
@@ -0,0 +1,64 @@
+package basicLibrary;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class lab3 {
+    public String analyzeWeather(int[][] arr){
+        int count = 1;
+        int max = 0;
```

Comment:
When hard coding a value it's typical to use all caps for the variable:
MAX = 0;

If there is more than one word in the variable name then you separate the words with _:
MAX_VALUE = 0;